docs/xgc.md

```diff
@@ -61,7 +61,9 @@ Options that change the behavior of the Garbage Collector (GC).
 
     <i class="fa fa-pencil-square-o" aria-hidden="true"></i> **Note: Linux on Z and z/OS**
 
-    This option is supported on IBM z14&trade; hardware running the following software:
+: This option is supported by all generations IBM Z&reg; hardware to enable pause-less GC with two modes of operation: hardware-based operations and software-based operations. This option enables software-based operations on IBM z13&trade; and earlier hardware; and it enables hardware-based operations on IBM z14&trade; and later hardware with supported software. One can switch from hardware-based operations to software-based operations with the ([softwareRangeCheckReadBarrier](xxgcsoftwareRangeCheckReadBarrier.md)) option. 
```

Comment:
See Alek's comments in the PR. We should not be documenting this. It should all be transparent to the user. You can leave everything here except the last sentence, so:

> This option is supported by all generations IBM Z&reg; hardware to enable pause-less GC with two modes of operation: hardware-based operations and software-based operations. This option enables software-based operations on IBM z13&trade; and earlier hardware; and it enables hardware-based operations on IBM z14&trade; and later hardware with supported software. ~One can switch from hardware-based operations to software-based operations with the ([softwareRangeCheckReadBarrier](xxgcsoftwareRangeCheckReadBarrier.md)) option.~

Reply:
Removed the aforementioned sentence and shortened this paragraph.